src/core/internal/reveal/reveal-frame.ts

```diff
@@ -140,8 +140,8 @@ class RevealFrame {
     document.body.append(this.#elementContainer);
 
     const sub = (data) => {
-      if (Object.prototype.hasOwnProperty.call(data, this.#record.token)) {
-        const responseValue = data[this.#record.token] as string;
+      if (Object.prototype.hasOwnProperty.call(data, this.#record.elementId)) {
```

Comment:
We need to handle backward compatibility as well, elementId will not be present in older versions of SDKs.